apps/package.json

```diff
@@ -34,8 +34,7 @@
     "storybook:deploy": "./script/deploy-storybook.sh"
   },
   "resolutions": {
-    "eshost": "github:pcardune/eshost#f3fa7fa",
-    "test262-compiler": "github:pcardune/test262-compiler#ffa0dff"
```

Comment:
Why were these existing package resolutions removed?

Reply:
Hmm, I'm not exactly sure. I ran `yarn add memoize-one` then `yarn` and this was the result. Should I change it back?

Reply:
Huh.. based on [this very old comment](https://github.com/code-dot-org/code-dot-org/pull/13974/files#r108005090), maybe not.  Maybe this was never needed, and the relevant issues that we forked for seem to have been resolved too.